src/components/Worker.tsx

```diff
@@ -1,16 +1,19 @@
-import React, { Component } from "react";
+import { Component } from "react";
 
-import { withDB } from "react-pouchdb/browser";
-
-import { GetMessageWorker } from "../workers/";
+import { GetMessageWorker } from "../workers";
 
 import { getUrl } from "../utils/api";
 
-class Worker extends Component {
-  working = false;
+type WorkerProps = {
```

Comment:
since we are in the `Worker` class file, it's safe to name this just `Props` (that's what we do in other react code)

```suggestion
type Props = {
```